mod1/ForTimesFor.java

```diff
@@ -0,0 +1,41 @@
+package mod1;
+
+import java.util.Scanner;
+
+public class ForTimesFor {
+    public static void main(String[] args) {
+        //scanner
+        Scanner whynot = new Scanner(System.in);
+        
+        //variables
+        int num, anser, imput, score;
+        String stringNum, stringImput;
+        score = 15;
+
+        //print
+        System.out.print("Which times table shall I recite? ");
+        stringNum = whynot.nextLine();
+        num = Integer.parseInt(stringNum);
+
+        //for loop
+        for (int num2 = 0; num2 < 15; num2++) {
```

Comment:
convention here is to use `i`,
so it would be `for (int i = 0; i < 15; i++)`

Reply:
how is that a problem

Reply:
its the same code but you want to use the variable `i` instead of `num2`